src/routes/middlewares/emailVerification.js

```diff
@@ -0,0 +1,50 @@
+import { Op } from 'sequelize';
+import models from '../../db/models';
+import { errorResponse } from '../../utils/apiResponse';
+
+const { User } = models;
+
+/**
+ * Check Email existence to prevent duplication
+ *
+ * @param {Object} req Express Request object
+ * @param {Object} res Express Response object
+ * @param {Object} next callback function
+ * @returns {Object} Error Response if validation fails
+ * @callback next pass control to the next function
+ */
+export const preventAccountDuplication = async (req, res, next) => {
```

Comment:
we can rename this to `checkDuplicateUser`.

Reply:
i have renamed it